torchrl/envs/transforms/transforms.py

```diff
@@ -1951,6 +1960,7 @@ class CatFrames(ObservationTransform):
             has to be written. Defaults to the value of `in_keys`.
         padding (str, optional): the padding method. One of ``"same"`` or ``"zeros"``.
             Defaults to ``"same"``, ie. the first value is uesd for padding.
+        as_inverse (bool, optional): if ``True``, the transform is applied as an inverse transform.
```

Comment:
Let me know what you think. Probably should have been an extra PR by itself. Is needed as we want to apply the frame stacking before we add the offline data to the replay buffer and not cat the frames after sampling -> needs to be inverse.

Reply:
Yep should be another PR